packages/components/select/src/select.vue

```diff
@@ -524,10 +524,10 @@ export default defineComponent({
         currentPlaceholder.value = ''
       }
       addResizeListener(selectWrapper.value as any, handleResize)
-      if (reference.value && reference.value.$el) {
-        const input = reference.value.input as HTMLInputElement
-        states.initialInputHeight = input.getBoundingClientRect().height
-      }
+      // if (reference.value && reference.value.$el) {
+      // const input = reference.value.input as HTMLInputElement
+      // states.initialInputHeight = input.getBoundingClientRect().height
+      // }
```

Comment:
Since we have already initiated the resize handler, there was no reason to do this manually.

Besides `input.height = input.height` seems nonsense to me.